src/msg/msg_legacy.cpp

```diff
@@ -297,6 +297,11 @@ sg_host_t MSG_host_self()
 {
   return sg_host_self();
 }
+
+double MSG_host_get_load(sg_host_t host)
```

Comment:
There were no need to extend MSG. The Java API could have used the S4U API directly. But your change is better than expected here, don't change it.